apps/codelab/src/app/codelabs/angular/component-tree/component-tree.component.html

```diff
@@ -121,14 +117,7 @@ <h2 i18n>
 
   <!--  PASSING DATA  -->
   <div *slide id="passing-data" milestone="component-tree">
-    <h1>
-      <ng-container i18n="@@componentTree">Component Tree</ng-container>
-      /
-      <ng-container i18n="@@passingDataParentChild"
-      >Passing Data Parent -> Child
-      </ng-container
-      >
-    </h1>
+    <codelab-breadcrumb i18n>Passing Data Parent -> Child</codelab-breadcrumb>
```

Comment:
Doesn't look good:
![image](https://user-images.githubusercontent.com/40883170/52531305-17a0d680-2d24-11e9-8220-1286be85183e.png)

May be something like: 
'Passing Data Parent to Child'
    or even 'Passing Parent Data to Child'